edg_core/Blocks.py

```diff
@@ -168,6 +202,8 @@ def __init__(self) -> None:
 
     self._elaboration_state = BlockElaborationState.init
 
+    self.description = DescriptionString("")   # additional string field to be displayed as part of the tooltip for blocks
```

Comment:
Would it make sense to distinguish between none and empty? For example
```suggestion
    self.description: Optional[DescriptionString] = None   # additional string field to be displayed as part of the tooltip for blocks
```